### PR TITLE
remove delta-mark files

### PR DESCRIPTION
**Pull request by lm-lily**
## What this patch does to fix the issue.
Remove delta-mark files.

## Link to any relevant issues or pull requests.
Issue #1066

### CODE REVIEW

**Comment by iizukak** [comment]
@lm-lily Thanks! This PR have some error on CI. Can you check?

File: tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ImageID,Source,LabelName,Confidence
```
Comment (iizukak) [inline tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv:1]: @lm-lily Which file read this file?

**Comment by lm-lily** [comment]
> @lm-lily
> Thank you for contributing.
> 
> `000026e7ee790996.jpg` is duplicate.
> And the license is unknown. Please consider to remove that file.

@iizukak 
All `000026e7ee790996.jpg` should be removed?
it was originally comes with 'tests/unit/fixtures/datasets/open_images_v4/images/train'.
I duplicated it to 'tests/unit/fixtures/datasets/open_images_v4/images/val/' and 'tests/unit/fixtures/datasets/open_images_v4/images/test' to solve the error when running 'make test-lmnet'.

File: tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv (hunk 1 above)
Comment (lm-lily) [reply tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv:1]: @iizukak 
If I remove this file, I get the following error:

```
===================================================== FAILURES =====================================================
__________________________________________ test_build_tfds_classification __________________________________________
[gw6] linux -- Python 3.6.3 /usr/local/pyenv/versions/3.6.3/envs/python3.6/bin/python

    def test_build_tfds_classification():
        environment.setup_test_environment()
    
        # Build TFDS Dataset
        config_file = "unit/fixtures/configs/for_build_tfds_classification.py"
>       run(config_file, overwrite=True)

unit/executor_tests/test_build_tfds.py:44: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../blueoil/cmd/build_tfds.py:96: in run
    builder.download_and_prepare()
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/api_utils.py:52: in disallow_positional_args_dec
    return fn(*args, **kwargs)
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py:287: in download_and_prepare
    download_config=download_config)
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py:937: in _download_and_prepare
    max_examples_per_split=download_config.max_examples_per_split,
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py:793: in _download_and_prepare
    for split_generator in self._split_generators(dl_manager):
../blueoil/utils/tfds_builders/classification.py:60: in _split_generators
    num_shards=self._num_shards(dataset),
../blueoil/utils/tfds_builders/classification.py:77: in _num_shards
    total_size = sum(image.nbytes for image, _ in dataset)
../blueoil/utils/tfds_builders/classification.py:77: in <genexpr>
    total_size = sum(image.nbytes for image, _ in dataset)
../blueoil/datasets/open_images_v4.py:261: in __getitem__
    files, labels = self.files_and_annotations
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <blueoil.utils.config.ClassificationDataset object at 0x7f8438094cf8>

    @property
    @functools.lru_cache(maxsize=None)
    def files_and_annotations(self):
        files = []
        annotations = []
>       with open(self.annotations_csv, 'r') as f:
E       FileNotFoundError: [Errno 2] No such file or directory: 'unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv'

../blueoil/datasets/open_images_v4.py:247: FileNotFoundError

```

`../blueoil/datasets/open_images_v4.py` calls to this file.

**Comment by iizukak** [comment]
@lm-lily 
I think you are confusing on some point.

First point. For classification, You don't need to use OpenImages format.
For classification, It's better to use CIfar10. Because it has more simple structure.
Using Cifar10 solve your problem with OpenImagesV4Classification.

Second point. For object detection, You are correct. Keep to use OpenImagesV4BoundingBoxBase.

Third point. And you do not need to remove original test image.
Please do not copy and paste it to save our repository size.

Maybe my answer is not enough. Please write question about non clear point.

Thanks.

**Comment by lm-lily** [comment]
@iizukak 
Iizuka san, thank you for clarifying the statements.

For classification, there is no CIFAR_10 dataset under `tests/unit/fixtures/datasets/`and we need to copy CIFAR_10 dataset to `tests/unit/fixtures/datasets/`.

There is another problem with CIFAR_10 dataset. There are 50000 num_of epoch for training and 10000 num_of_epoch for validation. It is far more than the num_of_epoch in other datasets of `tests/unit/fixtures/datasets/`. 

I found another dataset called `dummy_classification` and I have tested it for tfds.
I would like to know if it is ok to use dummy_classification as dataset, or it is better to use CIFAR_10 dataset for tfds test.

Thank you.

**Comment by lm-lily** [comment]
@iizukak 
May I have your review for this PR? 
I would like to close this PR soon.
Thousands thanks!

**Comment by iizukak** [comment]
@lm-lily Sorry, I'll review today 🙇

File: tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
+ImageID,Source,LabelName,Confidence
+000026e7ee790996,verification,/m/04hgtk,0
```
Comment (iizukak) [inline tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv:2]: @lm-lily Sorry again.
In my opnion, copy binary image `000026e7ee790996.jpg` is not good idea.
Is is impossible to use symlink or more small size dummy image?
681KBx2 is not reasonable for dummy image size.

Comment (lm-lily) [reply tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv:2]: @iizukak 
Yes, actually I am planning to remove `open_image_v4` as classification dataset.
I would like to use the originally available `dummy_classification` dataset to replace it.
If it is ok, I will commit the changes.

Comment (iizukak) [reply tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv:2]: @lm-lily Year, good to use `dummy_classification `

File: tests/unit/executor_tests/test_build_tfds.py [hunk 3]
```diff
@@ -48,8 +48,8 @@ def test_build_tfds_classification():
     train_run(config_file, expriment_id, recreate=True)
 
     # Check if the dataset was build correctly
-    train_data_num = 3
-    validation_data_num = 2
+    train_data_num = 27
+    validation_data_num = 3
```
Comment (a-hanamoto) [inline tests/unit/executor_tests/test_build_tfds.py:52]: Is this modification related to this PR?

File: tests/unit/executor_tests/test_build_tfds.py [hunk 4]
```diff
@@ -104,8 +104,8 @@ def test_build_tfds_object_detection():
     train_run(config_file, expriment_id, recreate=True)
 
     # Check if the dataset was build correctly
-    train_data_num = 3
-    validation_data_num = 2
+    train_data_num = 10
+    validation_data_num = 16
```
Comment (a-hanamoto) [inline tests/unit/executor_tests/test_build_tfds.py:108]: same as above

File: tests/unit/fixtures/configs/for_build_tfds_classification.py [hunk 5]
```diff
@@ -33,17 +33,16 @@
...
+    extend_dir = "dummy_classification"
 
 
 IS_DEBUG = False
 
 NETWORK_CLASS = LmnetV0Quantize
 DATASET_CLASS = ClassificationDataset
 
-IMAGE_SIZE = [128, 128]
+IMAGE_SIZE = [28, 28]
```
Comment (a-hanamoto) [inline tests/unit/fixtures/configs/for_build_tfds_classification.py:45]: same as above

File: tests/unit/executor_tests/test_build_tfds.py (hunk 3 above)
Comment (lm-lily) [reply tests/unit/executor_tests/test_build_tfds.py:52]: @a-hanamoto 
Yes, this is corresponding to the new chosen dataset that replaces the delta-mark dataset, for classification.

File: tests/unit/executor_tests/test_build_tfds.py (hunk 4 above)
Comment (lm-lily) [reply tests/unit/executor_tests/test_build_tfds.py:108]: This is the changes following the switching of delta-mark dataset to open_images dataset.

File: tests/unit/fixtures/configs/for_build_tfds_classification.py (hunk 5 above)
Comment (lm-lily) [reply tests/unit/fixtures/configs/for_build_tfds_classification.py:45]: This is also dataset dependent parameter. 
(28, 28) is the image size of delta-mark dataset.
